pages/Filters/create.vue

```diff
@@ -1,84 +1,41 @@
-                required>
-                <option value="all">{{ $t('allMarket') }}</option>
-                <option value="Ebay">eBay</option>;
-                <option value="kleinanzeigen">eBay Kleinanzeigen</option>
-              </select>
-            </div>
-          </div>
-
-          <!-- Search Value Field -->
-          <div>
-            <AneTextField :name="$t('srchval')" :placeholder="$t('srchPlace')" :label="$t('srchVal')"
-              v-model="filter.searchValue" />
-          </div>
-          <FiltersKeywordFilter :filter="filter" />
-
-          <!-- list all filters -->
-          <div v-for="(option, index) in filterStore.getFilterOptions">
-
-            <!-- Price Range -->
-
-            <!-- Radius -->
-            <FiltersRadiusRangeFilter :filter="filter" v-if="option.name == 'Radius'" />
-
-            <!-- Kleinanzeigen Kategorie -->
-            <FiltersKleinanzeigenCategoryFilter :filter="filter"
-              v-if="option.name === 'KleinanzeigenKategorie' && filter.marketplace == 'kleinanzeigen'"
-              :options="option.value.options" />
-
-            <FiltersBlacklistFilter :filter="filter" v-if="option.name == 'NotContainsKeyWord'" />
-          </div>
-
-          <NotificationSettingsFilter :filter="filter"></NotificationSettingsFilter>
-          <!-- Updated Blacklist Section with Confirmation -->
-
-          <!-- Notification Channels section -->
-          <div class="flex items-center justify-end space-x-4 pt-6 border-t border-slate-200 dark:border-slate-700">
-            <NuxtLink to="/filters"
-              class="px-4 py-2 rounded-lg border border-slate-300 dark:border-slate-600 text-slate-700 dark:text-slate-300 hover:bg-slate-50 dark:hover:bg-slate-700 transition-colors">
-              Cancel
-            </NuxtLink>
-            <button type="submit"
-              class="px-4 py-2 bg-indigo-600 text-white rounded-lg hover:bg-indigo-700 transition-colors">
-              {{ isNewFilter ? "Create Filter" : "Update Filter" }}
-            </button>
-          </div>
-        </form>
+  <CreateHeader :is-new-filter="isNewFilter" />
+  <UiDefaultContainer class="mb-6 p-6">
+    <form @submit.prevent="saveFilter" class="space-y-6">
+      <div class="grid grid-cols-1 md:grid-cols-2 gap-6">
+        <AneTextField :name="$t('filterName')" :placeholder="$t('nameEg')" :label="$t('filName')"
+          v-model="filter.name" />
+        <MarketplaceDropDown></MarketplaceDropDown>
       </div>
-    </div>
-  </div>
+
+      <AneTextField :name="$t('srchval')" :placeholder="$t('srchPlace')" :label="$t('srchVal')"
+        v-model="filter.searchValue" />
+
+      <FiltersKeywordFilter :model-value="filter.keywords" :label="$t('kewor')" :footer="$t('addKey')" />
+      <FiltersKeywordFilter :model-value="filter.blacklist" :label="$t('blackke')" :footer="$t('addblak')" />
+
+
+      <!-- Dont worry about this, this whole system needs to get reworked by akwav -->
```

Comment:
?

Reply:
That's handling categories in Kleinanzeigen which has not been also added to Ebay, the v-for is some ai code and I wasn't going to change it for now since iirc akwav was going to add the similar feature to Ebay which was going to be remade than